uplogic/nodes/actions/setvisibility.py

```diff
@@ -1,8 +1,12 @@
+from pickle import TRUE
```

Comment:
remove import from picke, TRUE is never used

Reply:
Sorry, fixed.